README.md

```diff
@@ -24,3 +24,4 @@ Visit a web friendly version at [https://suse-edge.github.io/charts/](https://su
 | Chart | Description |
 | - | - |
 | [Akri](charts/akri) | A Kubernetes Resource Interface for the Edge |
+| [Metal3-deploy](charts/metal3-deploy) | All of The Dependencies to Deploy Metal3 |
```

Comment:
Might be a dumb question, but why "Metal3-deploy" rather than just "Metal3" ?

Reply:
This was one that was a 50/50 decision, I just stuck with the name of the chart here. Will change it to just Metal3. Is the description for it alright?